app/assets/stylesheets/pensionsschemefrontend-app.scss

```diff
@@ -9,37 +9,37 @@
 
     .app-task-list__items {
         padding-left: 0;
+    }
```

Comment:
Is this curly bracket added and removed at line 46 intentionally ?

Reply:
@sqinnydave Might be best to answer this :)

Reply:
yea, this is all ok. is css syntax to isolate the class. otherwise we end up with extra padding on left of the tasklist, but we have slightly different setup/layout depending on toggles so was necessary to adjust.